projects/demo/src/modules/pipes/currency/currency.template.html

```diff
@@ -0,0 +1,71 @@
+<tui-doc-page
+    header="Currency"
+    package="ADDON-COMMERCE"
+    path="addon-commerce/pipes/currency.pipe.ts"
+>
+    <ng-template i18n pageTab>
+        Pipe for correct view in
```

Comment:
Hm, maybe: "Pipe for transforming number into money. It is usually used with <a ...>Input Number...